tutorials/epochs/baseline_regression.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+"""
+.. _ex-baseline_regression:
+
+====================================================================
+Baseline correction using generalized linear mixed models
```

Comment:
```suggestion
Regression-based baseline correction
```

I've come to regret the title of the paper a bit because it's not terribly clear for people who aren't deep in the statistical details. I've seen the method referred to as "regression-based" elsewhere and I think that's clear.

Two small notes:
1. "GLM" in the paper refers to "general linear model", which is the unifying theory for a lot of classical statistics and not "generalized linear model", which is an application of that theory. 
2. There are no mixed models in the example here, just classical linear regression. 😄